PWGDQ/Tasks/tableReader.cxx

```diff
@@ -991,7 +1002,7 @@ struct AnalysisSameEventPairing {
       if constexpr ((TPairType == pairTypeMuMu) && muonHasCov) {
         // LOGP(info, "mu1 collId = {}, mu2 collId = {}", t1.collisionId(), t2.collisionId());
         dileptonExtraList(t1.globalIndex(), t2.globalIndex(), VarManager::fgValues[VarManager::kVertexingTauz], VarManager::fgValues[VarManager::kVertexingLz], VarManager::fgValues[VarManager::kVertexingLxy]);
-        dileptonInfoList(t1.collisionId(), event.posX(), event.posY(), event.posZ());
+        dileptonInfoList(t1.globalIndex(), event.posX(), event.posY(), event.posZ());
```

Comment:
I don't remember this change. Was this something we discussed or it is also an artefact of the merging?